_guide_topics/opennurbs/what-is-opennurbs.md

```diff
@@ -13,40 +13,35 @@ keywords: ['openNURBS', 'overview']
 layout: toc-guide-page
 ---
 
- 
+
 ## Overview
 
 The openNURBS Initiative provides CAD, CAM, CAE, and computer graphics software developers the tools to accurately transfer 3-D geometry between applications.
 
 The openNURBS Toolkit consists of C++ source code for a library that will read and write openNURBS 3D model files (*.3dm*).  More than 400 software development teams and applications, including *Rhinoceros®*, exchange 3D models using the openNURBS (*.3dm*) file format.
 
+The openNURBS Toolkit reads and writes all Rhino 3DM files. Additionally, the openNURBS Toolkit provides NURBS evaluation tools and elementary geometric and 3D view manipulation tools.
+
+Unlike other open development initiatives, alliances, or consortia:
+
+- Commercial use is encouraged.
+- The tools, support, and membership are free.
+- There are no restrictions. Neither copyright nor copyleft restrictions apply.
+- No contribution of effort or technology is required from the members, although it is encouraged.
+
 The openNURBS Toolkit is intended for C++ and .NET programmers.  The toolkit includes complete source code to create a library that will read and write 3DM files.  The toolkit also includes source code for several example programs.
 
 ## Details
 
 The tools provided by openNURBS include:
 
-- C++ source code libraries to read and write the 3DM file format. The C++ compilers that are part of from Microsoft's Visual Studio and Apple's XCode are supported.
-- .NET source code to read and write the 3DM file format.
+- [Rhino3dmIO .NET Tookit]({{ site.baseurl }}/guides/opennurbs/what-is-rhino3dmio) .NET Package distributed as a Nuget package.  This contains assemblies to create and interpret OpenNurbs geometry.  Also, use this to read/write the Rhino 3DM file format.
+- [Rhino3dmIO .NET SDK source](https://github.com/mcneel/rhinocommon/wiki/Rhino3dmIO-Toolkit-(OpenNURBS-build)) - a .NET wrapper around OpenNurbs.
```

Comment:
I'd be wary of highlighting this, since we haven't released the current v6 version of the Rhino3dmIO source files.

Reply:
Yes, I am removing reference to the Rhino3DMio source for now.